libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -132,6 +132,11 @@ class AC_PrecLand
     // parameter var table
     static const struct AP_Param::GroupInfo var_info[];
 
+    // Allow Aux Function to enable/disable Copter PrecLand routine.
+    // This is different from the "Enable" parameter since we want the plnd data to be logged and the sensor to be working normally.
+    void plnd_active(bool active) { _plnd_active = active; }
```

Comment:
Please don't override like this.
```suggestion
    void set_active(bool active) { _plnd_active = active; }
```